prow/config/jobs/proxy-1.19.yaml

```diff
@@ -0,0 +1,1933 @@
+branches:
+- release-1.19
+env:
+- name: BUILD_WITH_CONTAINER
+  value: "0"
+image: gcr.io/istio-testing/build-tools-proxy:release-1.19-936207ec8823f21aa330a82c20649c6f9e2f7a22
+jobs:
+- command:
+  - ./prow/proxy-presubmit.sh
+  env:
+  - name: BUILD_WITH_CONTAINER
+    value: "0"
+  image: gcr.io/istio-testing/build-tools-proxy:master-936207ec8823f21aa330a82c20649c6f9e2f7a22
```

Comment:
Here's a line that has master that should have been updated to 1.19 per the issue. More lines in this file need the same update.